mesonbuild/dependencies/base.py

```diff
@@ -697,6 +697,11 @@ def _call_pkgbin(self, args, env=None):
         mlog.debug('PKG_CONFIG_PATH: ' + new_pkg_config_path)
         env['PKG_CONFIG_PATH'] = new_pkg_config_path
 
+        pkg_config_libdir_prop = self.env.properties[self.for_machine].get_pkg_config_libdir()
+        if pkg_config_libdir_prop:
+            new_pkg_config_libdir = ':'.join([p for p in pkg_config_libdir_prop])
```

Comment:
I know the same mistake is done above, but I think it should be using `os.pathsep` instead of `':'` because on Windows it's separated by `';'` IIRC.

Reply:
At least for pkg-config, the doc says it explicitly:

```
       PKG_CONFIG_PATH
              A colon-separated (on Windows, semicolon-separated) list of directories to search for .pc files.  The default directory will always be searched after searching the path; the default is
              libdir/pkgconfig:datadir/pkgconfig where libdir is the libdir for pkg-config and datadir is the datadir for pkg-config when it was installed.
```